package.json

```diff
@@ -12,8 +12,9 @@
     "postinstall": "npm prune && npm run build",
     "prepush": "opt --out prepush --exec 'npm run validate'",
     "start": "webpack-dev-server --config webpack/starterApp.js --devtool eval --progress --colors --hot --history-api-fallback --open",
-    "test": "NODE_ENV=test karma start",
-    "test:watch": "npm test -- --auto-watch --no-single-run --reporters progress",
+    "test": "jest",
+    "test:watch": "npm test -- --verbose --notify --watch",
```

Comment:
Interesting, I prefer the opposite.  I would like watch to give me specifics as that is what I would be using while doing TDD, and i'm fine with terse output for test.